File: pytools/examples/csharp/server/Protocol/AckSceneEnter.cs
using System.Collections;
using System.Collections.Generic;


public class AckSceneEnter
{
	private MsgScenePlayer _player;


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.WriteBuffer(this._player.GetBuffer());
		packet.Encode(Msg.P_ACK_SCENE_ENTER);
		return packet;
	}


	public MsgScenePlayer player
	{
		get { return this._player; }
		set { this._player = value; }
	}

}

File: pytools/examples/java/server/protocol/AckSceneEnter.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckSceneEnter
{
	private MsgScenePlayer player;


	public byte[] encode()
	{
		Packet packet = new Packet();
		packet.writeBytes(player.getBytes());
		return packet.encode((short)Msg.P_ACK_SCENE_ENTER);
	}



	public void setPlayer(MsgScenePlayer player)
	{
		this.player = player;
	}

}
